### DISCUSSION TITLE
Truncate Address

### DISCUSSION THREAD

**Discussion by wojciech-turek** [wevm/viem#1626] [Idea / Feature Request]
Hi,

I am willing to create a PR for additional function inside address utils directory that will truncate the address.
I feel like its such a common thing that everyone has to do in their dApp, it almost begs to be implemented in a lib like viem.

Additional motivation is that something similar already exists in other packages like `useDapp` in form of `shortenAddress`.

Please let me know your thoughts on the above.

**Comment by PaulRBerg** [comment]
A function that returns `0x`+`first 4 characters`+`...`+`last characters` would be great

e.g. `0x8DfE...7C01`
